README.md

```diff
@@ -179,6 +179,9 @@ In a Linux/Mac shell (or Git Bash on Windows), run
 curl -v -X POST '<webhook>'
 ```
 
+## Encryption
+Data encryption is in place at rest and in transit thanks to CosmosDB.
```

Comment:
Worth being clear that while in-transit encryption is _partially_ due to CosmosDB (it does communicate over an encrypted channel) it's arguably more important that we're forcing end users to encrypt-in-transit through use of HTTPS on our app service